python/dbusext/property.py

```diff
@@ -52,6 +52,7 @@ def __init__(self, dbus_interface, type_signature,
         # we use two underscores because dbus-python uses _dbus_interface to determine
         # whether an object is a dbus method
         self.__dbus_interface = dbus_interface
+        # TODO: use a different name instead, avoid double underscores
```

Comment:
Talked about it on Gitter, because of the comment just above :P Just have to pick another name.